templates/galaxy/config/user_preferences_extra_conf.yml.j2

```diff
@@ -67,6 +67,18 @@ preferences:
               required: False   
 {% endif %}
 
+    aarnet_filesender_account:
```

Comment:
could you please put this block between the {% if  %} {% endif %} so it's only on dev?